### PR TITLE
Added Kitsu support

### PR DESCRIPTION
**Pull request by kakasuarez**
Added support for [Kitsu](https://kitsu.io).
Screenshot
If you directly go to an anime's page, an error will be shown in the console but it should work **most** of the time regardless of the error.

### CODE REVIEW

File: kitsu/background.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+chrome.tabs.onUpdated.addListener(function (tabId, changeInfo, tab) {
+    if (changeInfo && changeInfo.status == 'complete' && tab.url.match("/kitsu.io/anime/")) {
+        inject(tabId);
+    }
+});
+
+function inject(tabId) {
+    chrome.tabs.executeScript(tabId, { file: "kitsu/script.js" })
+}
```
Comment (laxyapahuja) [inline kitsu/background.js:9]: unnecessary file

File: kitsu/script.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    </select> <button onclick="sort()">Sort</button>`
+    for (let i = (10 * pageCount); i < ((10 * pageCount) + endLimitDiscriminator()); i++) {
+        string += `<tr><td>
+    <div><a target="_blank" href="${torrents[i].link}">${torrents[i].title}</a>
+    <p style="float:right;">${torrents[i].size}</p></div>
+    <div><a style="float:left; margin-right: 5px;" target="_blank" href="${torrents[i].magnet}">Magnet</a>
+    <p style="float:left;">${torrents[i].seeders}↑</p><p style="float:left;">${torrents[i].leechers}↓</p>
+    <p style="float:right;">${torrents[i].time}</p></div></td></tr>`
+    }
+    string += `</tbody></table><center><p id="torrent-current-page">${pageCount + 1}/${pages()}</center><center><a onclick="pageBack();" style="margin-right: 5px; cursor: pointer;">Back</a><a onclick="pageNext();" style="cursor: pointer;">Next</a></center>`
```
Comment (laxyapahuja) [inline kitsu/script.js:32]: Try to make the UI better and more suited to Kitsu website UI. change the classes as well. This file is the exact copy of the file for MAL.

File: manifest.json [hunk 3]
```diff
@@ -16,12 +16,20 @@
...
             "js": ["anilist/content.js"]
-        }
+        },
+		{
+			"matches": ["https://kitsu.io/anime/*"],
+			"js": ["kitsu/content.js"]
+		}
     ],
-    "web_accessible_resources": ["myanimelist/script.js", "anilist/script.js"],
+    "background": {
```
Comment (laxyapahuja) [inline manifest.json:25]: remove this line

File: manifest.json [hunk 4]
```diff
@@ -16,12 +16,20 @@
         {
             "matches": ["https://anilist.co/anime/*"],
             "js": ["anilist/content.js"]
-        }
+        },
+		{
```
Comment (laxyapahuja) [inline manifest.json]: fix the indentation

File: kitsu/background.js (hunk 1 above)
Comment (kakasuarez) [reply kitsu/background.js:9]: This is not unnecessary. Without this, if you go on the Kitsu homepage and click on any anime, it doesn't work, leaving the only way it works being you having to type the anime's URL in the browser yourself or having to reload the page again after going there from the homepage.

File: kitsu/script.js (hunk 2 above)
Comment (kakasuarez) [reply kitsu/script.js:32]: I agree. Will do.

File: manifest.json (hunk 3 above)
Comment (kakasuarez) [reply manifest.json:25]: This is the same as above. We need a background script as well to make it work.

File: kitsu/background.js (hunk 1 above)
Comment (laxyapahuja) [reply kitsu/background.js:9]: oh okay didn't notice it has dynamic loading. okay, let this stay.

**Pull request merged**